### DISCUSSION TITLE
考虑加个后端吗

### DISCUSSION THREAD

**Discussion by sunist-c** [Arondight/Adachi-BOT#201] [Ideas]
故事起源于~~刁民~~群友想用bot背单词刷政治题，所以打算加个后端整合那些奇怪的功能（逃

**Comment by Arondight** [comment]
没理解你说的后端是什么，稍微展开一下说说？

**Reply by sunist-c** [reply]
就是用go/java一类写一个机器人后端专用于处理求签、抽卡（这个有呼声抽卡要百连）、强化圣遗物、今天吃派蒙、背单词、刷政治题的功能，我们初步使用目前这个框架发现性能有点......单纯背单词来说就几万个词汇加释义例句的量，大概10%左右会出现timeout没办法发送响应（）

**Comment by Arondight** [comment]
我目前没有这个考虑和需求……

**Comment by mark9804** [comment]
抽卡直接来一个90或者180连的缘之井这个功能我也在考虑，主要是因为“每个人抽十次实在是太容易把其他消息全刷下去了”

背单词和刷政治题的功能一开始应该是没考虑到这个功能的，另外几个维护者和我应该也没有题库……吧

我记得Arondight最近在剥离HTTP服务，加上这个后端变成开倒车了属于是

[SilveryStar/Adachi-BOT](https://github.com/SilveryStar/Adachi-BOT)的开发者现在好像也是大学生，他的V2框架也更倾向于前端+后端的结构，楼主也许可以和他商量一下共同维护
